apps/src/codebridge/ControlButtons/index.tsx

```diff
@@ -53,7 +55,9 @@ const ControlButtons: React.FunctionComponent = () => {
   const hasSubmitted = useAppSelector(
     state => getCurrentLevel(state)?.status === LevelStatus.submitted
   );
-  const disableRunAndTest = loading || (isPredictLevel && !hasPredictResponse);
+  const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
+  const disableRunAndTest =
+    loading || (!isStartMode && isPredictLevel && !hasPredictResponse);
```

Comment:
nit: This might be more readable if we make one more helper variable, like:

```
  const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
  const awaitingPredictSubmit = !isStartMode && isPredictLevel && !hasPredictResponse;
  const disableRunAndTest = loading || awaitingPredictSubmit;
```

Reply:
good call, will update